package.json

```diff
@@ -152,6 +152,7 @@
     "react-input-mask": "^2.0.4",
     "react-is": "^16.8.6",
     "react-movable": "^2.2.0",
+    "react-multi-ref": "^1.0.0",
```

Comment:
is this package required? seems like it's only a couple lines and is not widely used. writing our own code to do this will provide faster service to consumers in the case of bugs possibly resulting from it